src/Inviqa/Worldpay/Api/Request/PaymentService/Submit/Authorisation/AuthorisationOrder.php

```diff
@@ -28,7 +28,7 @@ public function __construct(
         OrderCode $orderCode,
         Description $description,
         Amount $amount,
-        PaymentDetails $paymentDetails,
+        PaymentDetailsApplePay $paymentDetails,
```

Comment:
The standard authorization order shouldn't have any reference to Apple Pay

Reply:
y find and replace fail